pkg/3scale/amp/component/system_options.go

```diff
@@ -60,6 +60,7 @@ type SystemOptions struct {
 
 	S3FileStorageOptions  *S3FileStorageOptions  `validate:"required_without=PvcFileStorageOptions"`
 	PvcFileStorageOptions *PVCFileStorageOptions `validate:"required_without=S3FileStorageOptions"`
+	MountPath             string
```

Comment:
Might be better to put this inside S3FileStorageOptions?

Reply:
done